src/pages/søknad/steg/oppsummering/Oppsummering.tsx

```diff
@@ -53,12 +104,7 @@ const Oppsummering = (props: { forrigeUrl: string; nesteUrl: string; avbrytUrl:
 
     return (
         <div className={sharedStyles.container}>
-            <form
-                onSubmit={(e) => {
-                    e.preventDefault();
-                    handleSubmit();
-                }}
-            >
+            <form onSubmit={form.handleSubmit((values) => handleSubmit(values))}>
```

Comment:
```suggestion
            <form onSubmit={form.handleSubmit(handleSubmit)}>
```
Kanskje rename `handleSubmit` til `submitHandler` også